features/convergedcloud/exec.config

```diff
@@ -3,12 +3,10 @@ set -Eeuo pipefail
 
 systemctl enable ignition-disable.service
 
-# crictl
-CRI_VERSION=1.26.1
+K8S_VERSION=v1.30.7
+K8S_VERSION_REPO="${K8S_VERSION%.*}"
 
-wget --show-progress --progress=bar:force -4 -P /usr/local/bin \
-  https://github.com/kubernetes-sigs/cri-tools/releases/download/v${CRI_VERSION}/crictl-v${CRI_VERSION}-linux-amd64.tar.gz
-
-rm -f /root/.wget-hsts
-
-tar -C /usr/local/bin -xf /usr/local/bin/crictl-v${CRI_VERSION}-linux-amd64.tar.gz
+gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg </builder/features/khost/release.key
+echo "deb [signed-by=/etc/apt/keyrings/kubernetes-apt-keyring.gpg] https://pkgs.k8s.io/core:/stable:/${K8S_VERSION_REPO}/deb/ /" | tee /etc/apt/sources.list.d/kubernetes.list
```

Comment:
Please use a different filename as used in `khost`. As it's for `cri-tools` you may name "key" and "list" files `kubernetes-cri-tools.*`.

Reply:
same as above